.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Question
+about: Ask a question about the ONNX.
+title: ''
+labels: 'question'
+assignees: ''
+
+ 
+
+---
+# Ask a Question
+
+### Question
+Explain your question here.
+
+### Further information
+- Relevant Area (*e.g. model usage, backend, best practices, shape_inference, version_converter, training, test, operators*): 
```

Comment:
some of these "relevant areas" might not make sense for the tutorials repo. (i.e. backend, because we would want them to go to the inference engine repo (pytorch / onnxruntime) directly?)

Reply:
also "operators" -> we'd want these in onnx/onnx

Reply:
Make sense. Thanks.